src/quacc/recipes/mlp/_base.py

```diff
@@ -71,6 +71,15 @@ def pick_calculator(
 
         calc = SevenNetCalculator(**kwargs)
 
+    elif method.lower() == "orb-models":
+        from orb_models import __version__
+        from orb_models.forcefield import pretrained
+        from orb_models.forcefield.calculator import ORBCalculator
+
+        orb_model = kwargs.pop("model", "orb_v2")
```

Comment:
Why are we popping the model keyword argument out? We shouldn't modify keyword arguments specified by the user. It would be better to do

```python
if "model" not in kwargs:
    kwargs["model"] = "orb_v2"
```

Reply:
We need to obtain the pre-trained model as an instance of `GraphRegressor` and pass it to `ORBCalculator`, instead of providing the model name as a string like we do in other calculators. Since the `model` won't be used elsewhere, I  popped it from the `kwargs`, but I realize that this may not be the best approach. In the recent update, I switched to using `kwargs.get()`.

Reply:
Ah, yes I like this better.